### ISSUE TITLE
Unable to access pagination with keyboard controls

### ISSUE THREAD

**Issue by sryeluri** [telerik/kendo-angular#2037]
Test Environment

OS: Windows 1809

Browser: Chrome 70.0.3538.77
Assistive Technology: NVDA 2018.2.1

Tools: NA

Steps:
1. Navigate to Application URL
2. Tab through the page to reach the bottom of the company list.
3. Observe that the pagination is not accessible via keyboard controls.

Actual Results: 
Keyboard only users are unable to use the pagination feature.

Expected Results:
All features available to mouse users must also be available via keyboard controls.

**Comment by ipeshev** [comment]
@sryeluri you can navigate grid with page up/page down, please refer to [https://www.telerik.com/kendo-angular-ui/components/grid/keyboard-navigation/](https://www.telerik.com/kendo-angular-ui/components/grid/keyboard-navigation/)
Working demo [here](https://stackblitz.com/edit/angular-set5lo?file=app/app.component.ts).

**Comment by tsvetomir** [comment]
I can confirm that the pager is not included in the tab sequence as per the [accessibility guidelines](https://www.w3.org/TR/wai-aria-practices/#grid) for data grids. By using the dedicated Page Up/Down shortcuts the focus can remain in the data cells.

A custom [pager template](https://www.telerik.com/kendo-angular-ui/components/grid/paging/#toc-pager-template) can be used to create alternative pager implementations.

**Comment by tsvetomir** [comment]
Reopening as the pager should be focusable when the Grid keyboard navigation is not enabled.
Apologies to @sryeluri for dismissing this bug report initially.

**Comment by dtopalov** [comment]
Fixed in v. 5.2.0 of the `@progress/kendo-angular-grid` package
